src/App.js

```diff
@@ -37,6 +38,7 @@ export default function App() {
         <StylesProvider injectFirst>
           <CssBaseline />
           <BrowserRouter>
+            <HandleRedirectTo />
```

Comment:
New component for handling ?redirectTo=... in the querystring and performing the desired redirect.